### PR TITLE
Security Overhaul [STAGE-2]

### PR DESCRIPTION
**Pull request by Gandalf2k15**
## About The Pull Request

Welcome to the security overhaul.

A brief summary of this PR is as follows: We aim to balance security over a less laser/instant win loadout. To achieve this we will be changing the kind of equipment security has access to. As of now, security has items that completely remove the fun in combat. This aims to reverse that, while still allowing them to be somewhat capable. 

We will be:

- [x] Swapping out the batong with the classic baton.
- [x] Swapping out the Energy Carbine with something more ballistic.
- [x] Swapping out the disabler for alternative weapons.
- [x] Changing the uniform up a little.

The general aesthetic I am now going for with security is the "peacekeeper" look. 

## New Uniform Set:
image

Officer:
image

Warden:
image

HoS:
image

Officers come with black beret variants too. All old clothes are of course still available as well. 

## New Armaments:
**G-17**
image
This is standard issue for all officers in place of the disabler.
You know the game with this one, it's chambered in 9mm, has a 16 round capacity, deals 10 damage+Stam.

**G-18**
image
This is the HoS's baby, it's the automatic version of the above.
Has a 3rnd burst and a 33 round capacity. Same damage.

**A-3 PCR**
image
The Automatik-3 Peacekeeper Cyclic Rifle is the replacement for the standard energy carbines in the armory. It uses WT550 ammo, including all the variants. 

**Peacekeeper Baton**
image
This is the replacement for the stun baton, all officers get one. It is worse than the classic baton, but should help with the issues we currently have with this being the go to. We want firefights, damn it!

## Why It's Good For The Game

We all enjoy SS13 for different reasons, and security has always been lacking somewhat in the department, we'll try to address this. It will evolve over time however I'd like to see how security copes with this.

## Changelog
:cl:
add: We've given our security department a new look. Hopefully this stuff won't attract bulls.
tweak: As a security officer, you are now equipped with an older style baton, we don't have the cash to keep sending you new state-of-the-art ones.
tweak: We've opted to use our surplus armory to stock up frontier stations. Expect varaing quality.
/:cl:

### CODE REVIEW

**Comment by MLGTASTICa** [comment]
Based , Redpilled and extremly big IQ . Stunbatons always proved to be an issue for fun . Are you aiming to turn sec more into eris style sec ? which fires rubber ballistics at antags as the most reliable way to arrest?

**Comment by BrotherHangyul** [comment]
Quick comment: stunbatons aren't one-hit wins anymore but they are still pretty busted will be honest.

Seems like a good change, all I ask is that you keep the HOS's uniform relatively similar because imo it still looks really nice

**Comment by SpookyTheFox** [comment]
I'm interested in seeing where this goes. Currently, stun batons are not strong at all though. They do not keep people down even after multiple hits right now. As an electronic item, having them disabled by EMPs was a good mechanic, but everything EMP related is pretty weak right now.

Ballistics will be interesting, especially if they are like before and cause brute damage, as that opens security to a mandatory response by our server's silicon policy if they use them.

**Comment by BrotherHangyul** [comment]
As a security main, I give this PR my blessing.

This looks really cool and is probably a good step towards making security more fun for sec-mains and antagonists alike.

👍

**Comment by YakumoChen** [comment]
Not a fan of the white. Looks pretty ugly in general and while it helps security stick out, it loses all sense of aesthetic and I feel like it really takes the teeth away from security as being anything but a kind of useless, foam-padded set of tard wranglers. 

The HoS especially I think looks best in black and I feel like this loses a lot of the sense of authority security asserts.

As I've said in other PRs, security being ballistic is a major nerf to a lot of antagonists (traitors especially) and a massive buff to a lot of other, already quite strong antagonists (changelings and blob gain massive benefit from stripping away security's lasers). That said I genuinely don't know how security is supposed to combat *anyone* without automatically reacting in a lethal manner. Given everyone moves at the same speed, all someone would have to do against an officer is run away most of the time, unless they happen to be really good at tossing bolas or something.

 This automatically goes against both escalation policy and against silicon's objectives to prevent harm and borgs are going to have to full time make sure security doesn't shoot **anything** ever since all of their arsenal is lethal now.

**Comment by YakumoChen** [comment]
Also it's kind of hilarious that you're trying to portray them as peacekeepers while also taking away any means of non-lethally taking anyone down. It's like if you took the white eggborg and only gave it a knife and a deagle to enforce the peace with. Disabler security at least made sense in this context.

**Comment by BrotherHangyul** [comment]
> Not a fan of the white. Looks pretty ugly in general and while it helps security stick out, it loses all sense of aesthetic and I feel like it really takes the teeth away from security as being anything but a kind of useless, foam-padded set of tard wranglers.
> 
> The HoS especially I think looks best in black and I feel like this loses a lot of the sense of authority security asserts.
> 
> As I've said in other PRs, security being ballistic is a major nerf to a lot of antagonists (traitors especially) and a massive buff to a lot of other, already quite strong antagonists (changelings and blob gain massive benefit from stripping away security's lasers). That said I genuinely don't know how security is supposed to combat _anyone_ without automatically reacting in a lethal manner. Given everyone moves at the same speed, all someone would have to do against an officer is run away most of the time, unless they happen to be really good at tossing bolas or something.
> 
> This automatically goes against both escalation policy and against silicon's objectives to prevent harm and borgs are going to have to full time make sure security doesn't shoot **anything** ever since all of their arsenal is lethal now.

You do know that the DE-sword has a ballistic deflect chance, right? Also, one item doesn't = a major nerf, it's a nerf to that one item. Everything else is mostly unaffected... actually, scratch that, DE-sword is the only item affected by this change. 

10 brute per shot is hardly lethal; yes it will conflict with silicons, but that makes for better RP if anything.

Laser carbines are still in-game, if I remember correctly. These rifles only replace the **energy carbines.** Blob and lings will remain counter-able.

**Comment by YakumoChen** [comment]
>You do know that the DE-sword has a ballistic deflect chance, right? Also, one item doesn't = a major nerf, it's a nerf to that one item. Everything else is mostly unaffected... actually, scratch that, DE-sword is the only item affected by this change.

The double sword reably _reflects_ energy weaponry, and has only a chance to _deflect_ (delete) any other projectiles. That's not the only item affected by this. This also makes EMPs a lot less usable (given nothing security is using is affected by it now).

>10 brute per shot is hardly lethal; yes it will conflict with silicons, but that makes for better RP if anything.

Anything more lethal than "not at all" is still lethal. That's the definition. Any projectile use by security now will automatically warrant a lethal response in return.

>Laser carbines are still in-game, if I remember correctly. These rifles only replace the energy carbines. Blob and lings will remain counter-able.

This takes all the lasers out of the armoury, leaving maybe only a couple on station (like cap's and HoS's if they aren't removed) and the rest needing to be ordered through cargo, which is inherently not reliable. Overall this presents an incredible boon to changelings and blob and any other antag who fares better against ballistic weaponry (infiltrators have better armor against ballistics, didn't you know?)


I question this change most because the visuals of laser weaponry are one of the strongest visible indicators in the game of projectiles and obstinately the main reason we use them. Disabler and laser fire are EXTREMELY visible and telling of their intent (the red ones kill you) and this strips a core facet of game design if we're going towards turning all projectile fire into the same bullet sprite.

The batons are, too, part of this. Stun batons are incredibly distinct in hand and you're replacing with with fairly generic grey lines.

**Comment by BrotherHangyul** [comment]
> The double sword reably _reflects_ energy weaponry, and has only a chance to _deflect_ (delete) any other projectiles. That's not the only item affected by this. This also makes EMPs a lot less usable (given nothing security is using is affected by it now).
> 
> >

EMP's deal brain damage and cause minor stamina damage, that should be enough for what they're worth.

I've quite literally looked at the code, there's no sign of changing the armoury. Coders like you are always pushing for more cargo interaction, this seems like a good way for security to interact with cargo.

If we're discussing infilitrators as in the admin-only spawned antagonist...

yeah no that's a stupid-ass point.

If we're discussing the infiltrator suit, then yes, so what? Current syndicate armour is focused around laser defence (though it often has similar amounts of ballistic defence regardless) and nobody's complained about that. 

You yourself mentioned lethality, antagonists can tell when something's escalating to lethal _because you yourself said that these rounds were lethal_. Baton is shit and won't do much.

**Comment by YakumoChen** [comment]
> EMP's deal brain damage and cause minor stamina damage, that should be enough for what they're worth.

That's a bug, if it's even confirmed to do that? EMPs have always been used to disable security's supply of typical weaponry and shut down borgs, and now they can only do half of that. The point of EMP is to shut down electronics, _obviously_

> I've quite literally looked at the code, there's no sign of changing the armoury. Coders like you are always pushing for more cargo interaction, this seems like a good way for security to interact with cargo.
image

🤔 

>You yourself mentioned lethality, antagonists can tell when something's escalating to lethal because you yourself said that these rounds were lethal. Baton is shit and won't do much.

Antagonists and security are NOT supposed to be jumping straight to lethal fighting, per escalation rules.

> Baton is shit and won't do much.

Hilariously this is actually a pretty good buff with the baton because the knockdown is instant (stun sticks have a delayed stun and are counterable with a chem in maint syringes) and it's a hard stun after only two hits. The inhands are also pretty difficult to see now. But what do I know, I'm just a security main, right?

**Comment by BrotherHangyul** [comment]
> > EMP's deal brain damage and cause minor stamina damage, that should be enough for what they're worth.
> 
> That's a bug, if it's even confirmed to do that? EMPs have always been used to disable security's supply of typical weaponry and shut down borgs, and now they can only do half of that. The point of EMP is to shut down electronics, _obviously_
> 
> > I've quite literally looked at the code, there's no sign of changing the armoury. Coders like you are always pushing for more cargo interaction, this seems like a good way for security to interact with cargo.
> > image
> 
> 🤔
> 
> > You yourself mentioned lethality, antagonists can tell when something's escalating to lethal because you yourself said that these rounds were lethal. Baton is shit and won't do much.
> 
> Antagonists and security are NOT supposed to be jumping straight to lethal fighting, per escalation rules.
> 
> > Baton is shit and won't do much.
> 
> Hilariously this is actually a pretty good buff with the baton because the knockdown is instant (stun sticks have a delayed stun and are counterable with a chem in maint syringes) and it's a hard stun after only two hits. The inhands are also pretty difficult to see now. But what do I know, I'm just a security main, right?

It LITERALLY says it's replacing ENERGY CARBINES. Not laser carbines.

Staff can decided on the use of rubbers.

Pretty sure it's not a hardstun after two hits if the coder specifies it's worse than the classic baton, which doesn't stamcrit after two hits. Due to reworks on our end, knockdown is worth jack-all.

I've been maining security since the rebase. I know quite a lot about security. Please check the fucking code and description before making an argument instead of cutting out the parts you like.

**Comment by YakumoChen** [comment]
> It LITERALLY says it's replacing ENERGY CARBINES. Not laser carbines.
Ah, yes, so removing all the non-lethal weapons from the armoury is also part of the solution I see. Stripping half of all the energy lethals from the armoury surely is not a major problem either right?

>Staff can decided on the use of rubbers.

What did he mean by this?

>Pretty sure it's not a hardstun after two hits if the coder specifies it's worse than the classic baton, which doesn't stamcrit after two hits. Due to reworks on our end, knockdown is worth jack-all.

I did read the code, and it does 55 stamina damage and an instant knockdown. The stunbaton does after a delayed knockdown, of which can be resisted. The classic baton is strictly better and also does not worry about cell charge. WOW!

>I've been maining security since the rebase. I know quite a lot about security. Please check the fucking code and description before making an argument instead of cutting out the parts you like.

Funny argument from someone continually cherrypicking half of my quotes.

**Comment by BrotherHangyul** [comment]
> > It LITERALLY says it's replacing ENERGY CARBINES. Not laser carbines.
> 
> Ah, yes, so removing all the non-lethal weapons from the armoury is also part of the solution I see. Stripping half of all the energy lethals from the armoury surely is not a major problem either right?
> 
> > Staff can decided on the use of rubbers.
> 
> What did he mean by this?
> 
> > Pretty sure it's not a hardstun after two hits if the coder specifies it's worse than the classic baton, which doesn't stamcrit after two hits. Due to reworks on our end, knockdown is worth jack-all.
> 
> I did read the code, and it does 55 stamina damage and an instant knockdown. The stunbaton does a mere 60 stamina after a delayed knockdown, of which can be resisted. The classic baton is strictly better and also does not worry about cell charge. WOW!
> 
> > I've been maining security since the rebase. I know quite a lot about security. Please check the fucking code and description before making an argument instead of cutting out the parts you like.
> 
> Funny argument from someone continually cherrypicking half of my quotes.

You were the one complaining there were no lasers. There are lasers.

It means staff can decide if rubbers are lethal or not.

/obj/item/melee/classic_baton/peacekeeper
	name = "peacekeeper baton"
	desc = "A metal-plastic bat, looks like it would REALLY hurt."
	icon = 'modular_skyrat/modules/sec_haul/icons/peacekeeper/peacekeeper_items.dmi'
	icon_state = "peacekeeper_baton"
	inhand_icon_state = "peacekeeper_baton"
	worn_icon_state = "classic_baton"
	lefthand_file = 'modular_skyrat/modules/sec_haul/icons/peacekeeper/baton/peacekeeper_baton_lefthand.dmi'
	righthand_file = 'modular_skyrat/modules/sec_haul/icons/peacekeeper/baton/peacekeeper_baton_righthand.dmi'
	slot_flags = ITEM_SLOT_BELT
	force = 10
	w_class = WEIGHT_CLASS_NORMAL
	cooldown = 60
	
	Where the fuck does it say 55 stam?

**Comment by YakumoChen** [comment]
> He doesn't know how to check code

**Comment by YakumoChen** [comment]
Also again, "10 brute" **IS** lethal. Any amount of actual lasting damage constitutes a lethal weapon.

You think tasers in real life are non-lethal weapons? No, they're 'less lethal' because they still fucking kill people.

**Comment by BrotherHangyul** [comment]
> Also again, "10 brute" **IS** lethal. Any amount of actual lasting damage constitutes a lethal weapon. Security should not be using any weapon that does non-stamina damage outside of a lethal encounter.
> 
> You think tasers in real life are non-lethal weapons? No, they're 'less lethal' because they still fucking kill people.

SS13 isn't real life. If you think otherwise, that's sad as fuck.

**Comment by YakumoChen** [comment]
I'm glad the only response you have left is calling me dumb, since I just dismantled all of your arguments on the matter. Have a nice night.

**Comment by BrotherHangyul** [comment]
> I'm glad the only response you have left is calling me dumb, since I just dismantled all of your arguments on the matter. Have a nice night.

If you want a fucking argument

10 brute plus stamina damage is hardly lethal. I really don't care about your definition, as long as it's not designed to kill, it's by definition not lethal. It's less than lethal. Flashes cause eye damage, pepperspray causes eye damage, flashbangs cause ear damage, and they're still used by sec. Really don't give a fuck about your own pompous definition.

The code says no stamina damage, unless you can get a direct quote I'm not going to believe some anti-sec coder like you.

Fuck off.

**Comment by YakumoChen** [comment]
>He doesn't know how to check code

"Hardly lethal" is still _lethal_. It literally is designed to kill people.

**Comment by YakumoChen** [comment]
I've been maining security since the rebase. I know quite a lot about security. Please check the fucking code and description before making an argument instead of cutting out the parts you like.

**Comment by BrotherHangyul** [comment]
> > He doesn't know how to check code
> 
> > 10 brute plus stamina damage is hardly lethal.
> 
> "Hardly lethal" is still _lethal_. It literally is designed to kill people.
> 
> > anti-sec coder
> 
> Ah yes, the security main, condemning this entire PR as anti-security trash. Obviously.

Rubbers are not designed to kill people, it's designed to cause a designated amount of pain to drop an attacker. Brute transfers into pain / stam-damage.

I've never seen you in-game, doubting you main security.

Yes, you're anti-sec. Anybody who immediately shits on any security-based PR is anti-sec.

**Comment by YakumoChen** [comment]
SS13 isn't real life. If you think otherwise, that's sad as fuck. 

Brute damage is still lethal in any context you put it in. It's bad for security in every manner for their default ranged option to be causing brute damage. There is almost no way to non-lethally arrest anyone in this manner if they run away.

> Yes, you're anti-sec. Anybody who immediately shits on any security-based PR is anti-sec.

Let me just PR removing security and call all the dissenters anti-sec. It's still a security-based PR, I don't know why you hate security so much? This PR is nothing but nerfs to security. It even says so in the description.  Please check the fucking code and description before making an argument instead of cutting out the parts you like.

**Comment by BrotherHangyul** [comment]
> SS13 isn't real life. If you think otherwise, that's sad as fuck.
> 
> Brute damage is still lethal in any context you put it in. It's bad for security in every manner for their default ranged option to be causing brute damage. There is almost no way to non-lethally arrest anyone in this manner if they run away.
> 
> > Yes, you're anti-sec. Anybody who immediately shits on any security-based PR is anti-sec.
> 
> Let me just PR removing security and call all the dissenters anti-sec. It's still a security-based PR, I don't know why you hate security so much? This PR is nothing but nerfs to security. It even says so in the description. Please check the fucking code and description before making an argument instead of cutting out the parts you like.

The desc says it's a balance, not a nerf. Learn the difference.

I'm not talking about real-life rubbers, SS13 rubbers literally are made to do stam damage and brute (equivalent of pain). Due to pain mechanics on skyrat, the brute also transfers into more stamina damage. Rubbers are inherently made to be less-than-lethal.

I've checked the code and there's nothing wrong. Grow the fuck up and learn how to use quotes properly.

**Comment by MLGTASTICa** [comment]
Please keep the discussion civil , also there is no pain system implementation currently , unless you consider stamina damage from brute a proper system .  10 Brute is too much for something to be non-lethal , only something under 7 brute is non-lethal , 10 Brute is the damage of getting hit with a toolbox.

**Comment by BrotherHangyul** [comment]
> Please keep the discussion civil , also there is no pain system implementation currently , unless you consider stamina damage from brute a proper system . 10 Brute is too much for something to be non-lethal , only something under 7 brute is non-lethal , 10 Brute is the damage of getting hit with a toolbox.

That's fair, might suggest a slight nerf to the brute then.

**Comment by YakumoChen** [comment]
It should just be stamina damage. Even oldbase USPs didn't do any brute on hit.

**Comment by YakumoChen** [comment]
> I've checked the code and there's nothing wrong. Grow the fuck up and learn how to use quotes properly.

No, you don't know how to check the code, because the peacekeeper baton is a child of the classic baton, and all the relevant info is in the code for the classic baton. _You do not know how to check code_ and I do not appreciate your hostile comments insinuating I do not know what I'm doing.

**Comment by BrotherHangyul** [comment]
> > I've checked the code and there's nothing wrong. Grow the fuck up and learn how to use quotes properly.
> 
> No, you don't know how to check the code, because the peacekeeper baton is a child of the classic baton, and all the relevant info is in the code for the classic baton. _You do not know how to check code_ and I do not appreciate your hostile comments insinuating I do not know what I'm doing.

You've literally been passive-aggressive throughout every single discussion, don't try to play innocent.

Classic baton, assuming that's the det's one, is actual shit. I've used it in-game. It's bad.

**Comment by YakumoChen** [comment]
I've only been reacting to your increasingly hostile and disparaging tone and using your comments back at you when you display an utter lack of knowledge on the subject. Calling me a code-illiterate idiot when I have 10 times as many code contributions than you on this website is laughable.

The classic baton does _instant knockdown_ and does almost as much stamina damage as the stun baton. The only difference here is the slightly longer delay between hits for the peacekeeper one. The detective, hilariously, will have the better baton than what security gets.
The stun baton is limited by charge, has its effects delayed by _two seconds_ and can have the stun resisted through ordinary, maint-findable means, whereas the classic baton variant is instant, does the stamina damage instantly, and can stunlock someone for eternity.

**Comment by BrotherHangyul** [comment]
> I've only been reacting to your increasingly hostile and disparaging tone and using your comments back at you when you display an utter lack of knowledge on the subject. Calling me a code-illiterate idiot when I have 10 times as many code contributions than you on this website is laughable.
> 
> The classic baton does _instant knockdown_ and does almost as much stamina damage as the stun baton. The only difference here is the slightly longer delay between hits for the peacekeeper one. The detective, hilariously, will have the better baton than what security gets.
> The stun baton is limited by charge, has its effects delayed by _two seconds_ and can have the stun resisted through ordinary, maint-findable means, whereas the classic baton variant is instant, does the stamina damage instantly, and can stunlock someone for eternity.



> I've only been reacting to your increasingly hostile and disparaging tone and using your comments back at you when you display an utter lack of knowledge on the subject. Calling me a code-illiterate idiot when I have 10 times as many code contributions than you on this website is laughable.
> 
> The classic baton does _instant knockdown_ and does almost as much stamina damage as the stun baton. The only difference here is the slightly longer delay between hits for the peacekeeper one. The detective, hilariously, will have the better baton than what security gets.
> The stun baton is limited by charge, has its effects delayed by _two seconds_ and can have the stun resisted through ordinary, maint-findable means, whereas the classic baton variant is instant, does the stamina damage instantly, and can stunlock someone for eternity.

This isn't citadel. Contributions there mean jack-all on this code.

You called *yourself* a code-illiterate idiot.

Then the coder can nerf the baton. That simple.

**Comment by Jeweledbug** [comment]
> 
> 
> > Not a fan of the white. Looks pretty ugly in general and while it helps security stick out, it loses all sense of aesthetic and I feel like it really takes the teeth away from security as being anything but a kind of useless, foam-padded set of tard wranglers.
> > The HoS especially I think looks best in black and I feel like this loses a lot of the sense of authority security asserts.
> > As I've said in other PRs, security being ballistic is a major nerf to a lot of antagonists (traitors especially) and a massive buff to a lot of other, already quite strong antagonists (changelings and blob gain massive benefit from stripping away security's lasers). That said I genuinely don't know how security is supposed to combat _anyone_ without automatically reacting in a lethal manner. Given everyone moves at the same speed, all someone would have to do against an officer is run away most of the time, unless they happen to be really good at tossing bolas or something.
> > This automatically goes against both escalation policy and against silicon's objectives to prevent harm and borgs are going to have to full time make sure security doesn't shoot **anything** ever since all of their arsenal is lethal now.
> 
> You do know that the DE-sword has a ballistic deflect chance, right? Also, one item doesn't = a major nerf, it's a nerf to that one item. Everything else is mostly unaffected... actually, scratch that, DE-sword is the only item affected by this change.
> 
> 10 brute per shot is hardly lethal; yes it will conflict with silicons, but that makes for better RP if anything.
> 
> Laser carbines are still in-game, if I remember correctly. These rifles only replace the **energy carbines.** Blob and lings will remain counter-able.

I'd like t note that the fact it deals brute damage WILL mean that crewsimov now affects their guns. Rubber is harm. So the change to rubber isn't quite that minimal.

**Comment by MLGTASTICa** [comment]
> I've only been reacting to your increasingly hostile and disparaging tone and using your comments back at you when you display an utter lack of knowledge on the subject. Calling me a code-illiterate idiot when I have 10 times as many code contributions than you on this website is laughable.
> 
> The classic baton does _instant knockdown_ and does almost as much stamina damage as the stun baton. The only difference here is the slightly longer delay between hits for the peacekeeper one. The detective, hilariously, will have the better baton than what security gets.
> The stun baton is limited by charge, has its effects delayed by _two seconds_ and can have the stun resisted through ordinary, maint-findable means, whereas the classic baton variant is instant, does the stamina damage instantly, and can stunlock someone for eternity.

The delay is not a slightly longer delay , its actually a very big delay difference.

**Comment by Gandalf2k15** [comment]
> I've only been reacting to your increasingly hostile and disparaging tone and using your comments back at you when you display an utter lack of knowledge on the subject. Calling me a code-illiterate idiot when I have 10 times as many code contributions than you on this website is laughable.
> 
> The classic baton does _instant knockdown_ and does almost as much stamina damage as the stun baton. The only difference here is the slightly longer delay between hits for the peacekeeper one. The detective, hilariously, will have the better baton than what security gets.
> The stun baton is limited by charge, has its effects delayed by _two seconds_ and can have the stun resisted through ordinary, maint-findable means, whereas the classic baton variant is instant, does the stamina damage instantly, and can stunlock someone for eternity.

I thought you said this was a nerf to security?

**Comment by Gandalf2k15** [comment]
I'm going to say this; what happens here is based upon what the future ideas of this server are. If you haven't noticed we've also been adding more high intensity gamemodes. This aims to counteract those changes in a flavoursome way. This is what the host has asked for for a good long while. So while I understand the arguing, it is ultimately in vein.

**Comment by BrotherHangyul** [comment]
Gandalf, only criticism I have right now is maybe the rubbers should do either less brute or no brute due to silicosn being silicons, but other than that, I agree with the changes.

**Comment by H0gberg** [comment]
Batong change is good, I think the guns need a lil' testing for their lethality. As for the outfit, maybe keeping them green or blue while the HOS stays black would be nice, white is a little blinding IMO. Also, the arguing on this PR is so fucking dumb, just speak like normal human beings you apes.

**Comment by tf-4** [comment]
> Batong change is good, I think the guns need a lil' testing for their lethality. As for the outfit, maybe keeping them green or blue while the HOS stays black would be nice, white is a little blinding IMO. Also, the arguing on this PR is so fucking dumb, just speak like normal human beings you apes.

Pogberg is correct again - damage on the guns is not a good idea, as dealing with silicons that lock you down every single round because you fired a gun you have no alternatives for is not a fun time.

I'm also not sure why disablers need to be removed here, is there no way for them to coexist? Maybe a small disabler nerf to bring them in line, then it's preference of lasers vs ballistics.

**Comment by Zenitheevee** [comment]
a few... Personal words i'd like to  say...
1: The white looks... Honestly... not horrible, But it doesn't look good... I'd almost suggest more of the silverish-grey that the new peacekeeper shotgun has. 
2: The baton is.. Honestly more going to get tossed because its ugly as all get-out. I'd personally suggest something along side the look of a stylized billy-club, but as a non-sec player i'm not going to see the sprites all that much either way.
3: *Maybe* keep the HoS' current gun, In some way or another, Available to people. A gun that has disable, kill and ion is a really powerful tool, especially considering that a lot of energyguns just got locked behind crafting with energy carbines (Which, fun fact, Nerfed charlie ever-so-slightly)

**Comment by BrotherHangyul** [comment]
> a few... Personal words i'd like to say...
> 1: The white looks... Honestly... not horrible, But it doesn't look good... I'd almost suggest more of the silverish-grey that the new peacekeeper shotgun has.
> 2: The baton is.. Honestly more going to get tossed because its ugly as all get-out. I'd personally suggest something along side the look of a stylized billy-club, but as a non-sec player i'm not going to see the sprites all that much either way.
> 3: _Maybe_ keep the HoS' current gun, In some way or another, Available to people. A gun that has disable, kill and ion is a really powerful tool, especially considering that a lot of energyguns just got locked behind crafting with energy carbines (Which, fun fact, Nerfed charlie ever-so-slightly)

Pretty sure the Glock-18 replaces the HOS's roundstart carbine rather than their personal gun.

**Comment by ayebee123** [comment]
**_HOOOOOOOOOOOOOOOOOOOOOOOOOOOOOO_** WHAT THE FUCK DID I JUST READ
okay so this is coming from the QM main, granted on other servers i actually did play sec frequently and somewhat understand what makes it fun so here we go

first off, the uniforms. good god you literally took away all of the soul from them by making them pure white. by any chance were you a /tg/ coder? they’re good at that. if your intention wasn’t to make the uniforms look soulless, what the fuck was it? even though we’re on an ERP server, you don’t need to make everything look like it’s glazed in cum.

the weapons. honest to god, i literally cannot fathom what you were thinking. first off, the glock. or as i'd like to call it, "the shitty disabler replacement." **_because it is._** if anyone is wearing decent armor with bullet protection, their damage is **_heavily_** reduced now. let's say, you're up against a traitor who had the wise idea to buy a blood-red for combat. they don't have _any_ syndicate weapons, but only the blood-red. they just have a toolbox. easy fight, right? nah. bullet 50%, so that's a complete 5 damage reduction to the pistol. by the way, i've seen the vid of how effective it is against the straight jacket guy. **_15_** shots to down a single person. with no armor. blood red, that's now 30, due to the 50% bullet reduction. let's say the syndicate has a toolbox, aiming for the chest. toolbox does 12 damage. armor vest has 35% melee resist, 7.8 damage per hit. considering the syndie only needs to land 13 hits to down the officer opposed to the fucking 30 shots, it's safe to say the syndie wins. by the way, that's only body shots. i'm pretty sure if they were to hit the head, it's even faster, as the helmet has less protection if i'm not mistaken. hos's gun is the same thing but 3 burst and 33 rounds, so it just unloads bullets faster. neat. and still potentially lethal to the crew if anything. A-3 PCR is just a reskinned WT from what i understand, so i don't care. let's move on.

so i've established the guns already kinda blow giant futa goat dick, so what's the big problem? two things. one: antags will get it easier since if they're smart and armor up + get a syndicate weapon, sec essentially goes bye-bye and they're rendered pointless a few minutes into the round. the second problem. 

_**these are all lethal weapons.**_
so lets say security has to deal with a rowdy, drunken crewmember who's fucking shit up. they have a broken bottle and willing to use it against the officers. how will they deal with the situation?
1. use their shitty batons and risk getting damaged or worse if they're unrobust as shit
2. act like your average american cop and gun down the poor guy
i'd have much less of a problem if the glock was a non-lethal weapon with lethal rounds optional, but it looks like lethal is the main option since that's the default ammo type. so of course, as the QM main, i'm going to order sec laser carbines.

_**I'll do it so the 20 dogshit sec mains on this server don't end up ruining someone's rounds with lethals on green alert.**_

in short, don't make the sec uniforms cum, don't give sec lethal weapons round start, buff the baton, add back disablers or some fucking non-lethal weaponry.

alternatively, make a section of the armory with lethal weapons that opens on certain alerts. that'd be better.

-your local QM main.

**Comment by Gandalf2k15** [comment]
> **_HOOOOOOOOOOOOOOOOOOOOOOOOOOOOOO_** WHAT THE FUCK DID I JUST READ
> okay so this is coming from the QM main, granted on other servers i actually did play sec frequently and somewhat understand what makes it fun so here we go
> 
> first off, the uniforms. good god you literally took away all of the soul from them by making them pure white. by any chance were you a /tg/ coder? they’re good at that. if your intention wasn’t to make the uniforms look soulless, what the fuck was it? even though we’re on an ERP server, you don’t need to make everything look like it’s glazed in cum.
> 
> the weapons. honest to god, i literally cannot fathom what you were thinking. first off, the glock. or as i'd like to call it, "the shitty disabler replacement." **_because it is._** if anyone is wearing decent armor with bullet protection, their damage is **_heavily_** reduced now. let's say, you're up against a traitor who had the wise idea to buy a blood-red for combat. they don't have _any_ syndicate weapons, but only the blood-red. they just have a toolbox. easy fight, right? nah. bullet 50%, so that's a complete 5 damage reduction to the pistol. by the way, i've seen the vid of how effective it is against the straight jacket guy. **_15_** shots to down a single person. with no armor. blood red, that's now 30, due to the 50% bullet reduction. let's say the syndicate has a toolbox, aiming for the chest. toolbox does 12 damage. armor vest has 35% melee resist, 7.8 damage per hit. considering the syndie only needs to land 13 hits to down the officer opposed to the fucking 30 shots, it's safe to say the syndie wins. by the way, that's only body shots. i'm pretty sure if they were to hit the head, it's even faster, as the helmet has less protection if i'm not mistaken. hos's gun is the same thing but 3 burst and 33 rounds, so it just unloads bullets faster. neat. and still potentially lethal to the crew if anything. A-3 PCR is just a reskinned WT from what i understand, so i don't care. let's move on.
> 
> so i've established the guns already kinda blow giant futa goat dick, so what's the big problem? two things. one: antags will get it easier since if they're smart and armor up + get a syndicate weapon, sec essentially goes bye-bye and they're rendered pointless a few minutes into the round. the second problem.
> 
> _**these are all lethal weapons.**_
> so lets say security has to deal with a rowdy, drunken crewmember who's fucking shit up. they have a broken bottle and willing to use it against the officers. how will they deal with the situation?
> 
> 1. use their shitty batons and risk getting damaged or worse if they're unrobust as shit
> 2. act like your average american cop and gun down the poor guy
>    i'd have much less of a problem if the glock was a non-lethal weapon with lethal rounds optional, but it looks like lethal is the main option since that's the default ammo type. so of course, as the QM main, i'm going to order sec laser carbines.
> 
> _**I'll do it so the 20 dogshit sec mains on this server don't end up ruining someone's rounds with lethals on green alert.**_
> 
> in short, don't make the sec uniforms cum, don't give sec lethal weapons round start, buff the baton, add back disablers or some fucking non-lethal weaponry.
> 
> alternatively, make a section of the armory with lethal weapons that opens on certain alerts. that'd be better.
> 
> -your local QM main.

I think you get "the least useful comment of the year" award.

**Comment by GoldenAlpharex** [comment]
I must say, I'm personally not a fan of the white look in the slightest. I don't really know why you'd want to change Security's looks, I'd say they look really good as-is. The color scheme doesn't match the department colors, so it makes very little sense and comes out of nowhere.

Secondly, I... Don't know what to think about security using lethals, to be honest. The whole point of using laser weapons was so Security could do some non-lethal takedowns via the whole mechanic of stamina damage, instead of relying on people going to crit or getting hard-stunned by tasers.

If it was up entirely to me, I'd separate this in a couple different PRs, especially the new look for security, which doesn't really fit the rest of the PR. I'm ready to try security ballistics, but that'll need a lot of balancing, however I'm really not looking forward to having stormtroopers as security.

**Comment by BrotherHangyul** [comment]
PR gone.

What.

**Comment by MLGTASTICa** [comment]
You're splitting this ? please tell me you are.

**Comment by hugmeorelse** [comment]
I'm going to start removing people who choose to continue to be toxic. Consider this a final warning on the subject.

If you have a disagreement, speak like a normal human being. If you cannot be decent in public, you have no place in this community.
